src/chocolatey/GetChocolatey.cs

```diff
@@ -49,8 +49,9 @@ public class GetChocolatey
 
         public GetChocolatey()
         {
+            Log4NetAppenderConfiguration.configure();

```

Comment:
Actually, if no logger is configured, how do you get this to work? 
It's set to NullLogger by default - https://github.com/chocolatey/choco/blob/master/src/chocolatey/infrastructure/logging/Log.cs#L27-L37